dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -1126,6 +1138,7 @@ export const decidePalette = (format: ArticleFormat): Palette => {
 			richLink: fillRichLink(format),
 			quoteIcon: fillQuoteIcon(format),
 			blockquoteIcon: fillBlockquoteIcon(format),
+			twitterHandleBelowDesktop: fillTwitterHandleBelowDesktop(format),
```

Comment:
There's no `fill.twitterHandle` here, is that because the designs don't need it?